randovania/games/common/prime_family/gui/dolphin_connector_prompt_dialog.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from typing import Any
+
+from PySide6 import QtWidgets
+
+from randovania.gui.dialog.text_prompt_dialog import TextPromptDialog
+from randovania.gui.lib import async_dialog
+
+
+class DolphinConnectorPromptDialog(TextPromptDialog):
+    """Similar to ConnectorPromptDialog, but with more options."""
+
+    def __init__(self, **kwargs: Any):
+        super().__init__(**kwargs)
+
+        # Top line: Launch & Autodetect radio buttons
+        self.launch = QtWidgets.QRadioButton("Launch", self)
+        self.launch.setToolTip(
+            "Run a custom command under Randovania's control, then connect to it. "
+            "This may be needed to work on Linux systems with default security settings."
+        )
+        self.autodetect = QtWidgets.QRadioButton("Autodetect", self)
+        self.autodetect.setToolTip("Attempt to find and connect to an already running Dolphin.")
+        self.gridLayout.addWidget(self.autodetect, 1, 0, 1, 1)
+        self.gridLayout.addWidget(self.launch, 1, 1, 1, 2)
+
+        # Next line, Run Command radio button, text field
+        self.cmd_label = QtWidgets.QLabel("Command", self)
+        self.prompt_edit.setToolTip(
```

Comment:
do not use tooltips for labels. Absolutely not intuitive.
Use a question-mark/info-icon button instead. See plasma-settings' Display KCM as well as our preset-settings-item-tab for reference
![grafik](https://github.com/user-attachments/assets/32bf5e2b-4e3f-44ce-ae3a-f1de20693094)

Reply:
I was going to actually ask that this dialog is made with an UI file. There's way too much widget creation here that isn't dynamic.